components/MapList/index.js

```diff
@@ -72,6 +75,8 @@ function MapList({
 
   const featureCounts = useFeatureCounts({ geojson, filters });
 
+  const totalFeatureCount = useTotalFeatureCount(featureCounts);
```

Comment:
this hook looks great, but in this case i think we might be able to just check if `geojson?.features.length > 0`, since `geojson` contains the full feature array before any filters are applied?

Reply:
It does contain the full features array which means that it also returns all of the signals with communication issues since those are still in the dataset. For example the length of total feature count right now is 1 but geojson features length is 31.

Reply:
ah, i see. should be filtered in the socrata API call if we don't want them?